sections/scenario_end.tex

```diff
@@ -16,13 +16,22 @@
 If you are Eliminated, you may still participate in the game by controlling Neutral Units.\par
 \note{4}{If you Eliminate all enemy Factions, you immediately win the Scenario.}\par
 
-\columnbreak
-
 In \textbf{Clash} Scenarios with three or more players, collecting \textbf{a Faction Cube} from every enemy player immediately wins you the game.
 Other Scenario specific rules may also modify the outcome of collecting Faction Cubes.
 
+\columnbreak
+
+After finishing a \textbf{Solo Campaign} scenario, reset your Hero's Experience Level to 1, and prepare the starting deck for the next scenario of the campaign.
+It will consist of:
+\begin{itemize}
+  \item all the \hyperlink{Statistic}{Statistic} cards from your deck,
+  \item the level 1 \hyperlink{Specialty}{Specialty} card,
+  \item 5 other non-Specialty cards of your choice from your deck.
+\end{itemize}
+Skip steps 11-13 of Setup for the next scenario of the campaign.
+
 \vspace*{\fill}
 \hspace{-6em}
-\includegraphics[width=2\linewidth]{\art/medusa.png}
+\includegraphics[width=1.8\linewidth]{\art/medusa.png}
```

Comment:
![image](https://github.com/Heegu-sama/Homm3BG/assets/5611195/768b984e-66bd-4c93-b4db-10d50eb47c44)

Reply:
Let me push an updated Medusa and see how it works. Hang on.

Reply:
There. Now restore `2` and see how it looks.

Reply:
I don't know what you changed, but please build it, and see how it doesn't work

Reply:
PTAL

Reply:
A little furgher to the right

Reply:
moved